requirements.txt

```diff
@@ -196,3 +200,7 @@ psycopg2-binary==2.8.3 \
  --hash=sha256:d305313c5a9695f40c46294d4315ed3a07c7d2b55e48a9010dad7db7a66c8b7f\
  --hash=sha256:19d013e7b0817087517a4b3cab39c084d78898369e5c46258aab7be4f233d6a1
 
+# required by django-storages 1.11.1
```

Comment:
Looking at `poetry.lock` seems like this dependency is not needed

Reply:
poetry.lock has got django-storages package so I think it's needed?

Reply:
Sorry, I'm talking about `python-dateutil` below